src/docs/style/logo.mdx

```diff
@@ -0,0 +1,267 @@
+
+You should also verify that your product logo is distinguished enough from other existing Eaton product logos.
+
+Glyphs are usually aligned to the keyline (visible in the template) to avoid bleeding beyond the safe zone.
+
+<ImageGrid images={[GlyphKeyline]} captionsUnderImages={[`Align glyphs to the icon template keyline.`]} regularWidth />
+
+Once the base glyph is decided, you can be creative and add variations to the glyph to make it look more designed.
+
+<ImageGrid
+    images={[PolishGlyph1, PolishGlyph2]}
+    captionsUnderImages={[
+        `Smart Energy Manager modified the glyph shape to accommodate for a bolt glyph inside.`,
+        `Predict Pulse modified the glyph size, opacity, and image gradient.`,
+    ]}
+/>
+
+## Background
+
+Most Eaton app logos use [the primary blue](/style/color?category=ui&name=blue&weight=500) as the base background color. Some also select the background color from one of the [branding colors](/style/color#branding-colors) to stay consistent with colors in their product marketing.
+
+<ImageGrid
+    images={[Background1]}
+    captionsUnderImages={[`Use the primary blue or one of the branding colors as a foundation for the background.`]}
+    regularWidth
+/>
+
+<ImageGrid
+    images={[Background2, Background3]}
+    captionsUnderImages={[
+        `DONT: Do not use a photo in the logo.`,
+        `DONT: Do not use a screenshot or app UI in the logo.`,
+    ]}
+/>
+
+<ImageGrid
+    images={[BackgroundGradient1, BackgroundGradient2, BackgroundGradient3]}
+    captionsUnderImages={[
+        `DO: Most Eaton apps choose to add a subtle corner linear gradient to their app logo background.`,
+        `Apps may also consider using a centered radial gradient, especially when the glyph has a round shape.`,
+        `A monochromatic background is acceptable.`,
+    ]}
+    gridImageProps={{ sm: 6, md: 4 }}
+/>
+
+## Bottom Badge
+
+The bottom badges are not meant to be modified. You are expected to leave the bottom badge as it is in the logo templates.
+
+<ImageGrid
+    images={[BottomBadge1, BottomBadge2, BottomBadge3, BottomBadge4, BottomBadge5]}
+    captionsUnderImages={[
+        `DO: Use the default badge as it is.`,
+        `DONT: Don't modify the badge color.`,
+        `DONT: Don't modify the badge logo color.`,
+        `DONT: Don't modify the badge size or the logo size.`,
+        `DONT: Don't modify the badge effects or add shadows.`,
+    ]}
+    gridImageProps={{ sm: 6, md: 4 }}
+/>
+
+## Export
+
+When exporting to different platforms, ensure that your logo design is following the platform-specific requirement [as listed below](#logo-requirements-by-platforms).
+
+When exporting to specific resolutions it is sometimes helpful to tweak vector shapes so that they fall on exact pixels, a process also known as "pixel hinting". This helps maintain a “crisp” look and avoids the blurriness that may occur when edges are in between pixels.
```

Comment:
```suggestion
When exporting for specific resolutions it is sometimes helpful to tweak vector shapes so that they fall on exact pixels, a process also known as "pixel hinting". This helps maintain a “crisp” look and avoids the blurriness that may occur when edges are in between pixels.
```